lib/assets/javascripts/jquery.mentionsInput.js

```diff
@@ -95,6 +96,7 @@
       elmInputBox.attr('data-mentions-input', 'true');
       elmInputBox.bind('keydown', onInputBoxKeyDown);
       elmInputBox.bind('keypress', onInputBoxKeyPress);
+      elmInputBox.bind('paste',onInputBoxPaste);
```

Comment:
Please note that we enforce the styleguides and I made an exception here since this file wasn't originally written by us and #6537 will remove the file from the develop branch. Your fix will be part of the 0.5.6.0 release. (not 0.5.5.0, which will be released this weekend)